api/hypershift/v1alpha1/capi_types.go

```diff
@@ -1,14 +0,0 @@
-package v1alpha1
```

Comment:
Why is this only dropped from the alpha? hwo about beta?

Reply:
I removed both alpha and beta, the thing is that GitHub shows that as "renamed" from `api/hypershift/v1alpha1/capi_types.go` to  `support/api/capi_types.go`